Signal/src/ViewControllers/ConversationView/Cells/OWSExpirationTimerView.h

```diff
@@ -6,11 +6,19 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+extern const CGFloat kExpirationTimerViewSize;
+
 @interface OWSExpirationTimerView : UIView
```

Comment:
This class has been heavily reworked:

* Is now auto layout friendly.
* Now can be used at any size.
* Better handling of edge cases.